### PR TITLE
Fixes #26 - Move CI from Azure to GH Actions

### LINKED ISSUE #26: Move the CI from Azure to GH Actions
**Is your feature request related to a problem? Please describe.**
Azure is on a different platform, and it is a hassle for new process groups to make changes to it.

**Describe the solution you'd like**
To move the CI to Github Actions, to keep everything on one platform

**Describe alternatives you've considered**
none

**Additional context**
nothing

Related to https://github.com/aau-giraf/web-api/issues/66

### PR DESCRIPTION
**Pull request by mathiasandresen**
fixes #26

### CODE REVIEW

File: .github/workflows/build-and-test.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+          channel: 'stable'
+      - run: flutter pub get
+        name: Get dependencies
+
+      - run: flutter analyze
+        name: Linter
+      - run: flutter test
+        name: Tests
+
+  build-and-test-ios:
```
Comment (frederikspang) [inline .github/workflows/build-and-test.yml]: Remove this block, as it's unnecessary.

File: .github/workflows/build-and-test.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build-and-test:
+    name: Android on ${{ matrix.os }}
```
Comment (frederikspang) [inline .github/workflows/build-and-test.yml]: Det er ikke android; Fix navn

File: .github/workflows/build-and-test.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
```
Comment (frederikspang) [inline .github/workflows/build-and-test.yml]: skal vi teste på 16.04 også?

**Comment by Seraph2** [comment]
Why is it Java version 12? Java 11 is the latest LTS.

File: .github/workflows/lint-and-test.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    name: Lint and test package on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '12.x'
```
Comment (Seraph2) [inline .github/workflows/lint-and-test.yml]: Why is it Java version 12? Java 11 is the latest LTS.

Comment (TobiasNielsen) [reply .github/workflows/lint-and-test.yml]: Also recommend using the LTS version 11.

**Pull request merged**